src/components/App/App.test.js

```diff
@@ -8,9 +8,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import { renderWithRedux } from '../../setupTests';
+import { act, screen } from '@testing-library/react';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+jest.mock('../../janus-api');
```

Comment:
I was going to use the [`automock`](https://jestjs.io/docs/en/configuration.html#automock-boolean) option instead of mocking it explicitly per test file, but unfortunately CRA does not allow it without ejecting.